src/KPI.jsx

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+
+const Box = styled.section`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  list-style: none;
+  background: linear-gradient(#1486e4, #62b4f7);
+  color: white;
+  width: 10rem;
+  padding: 1rem;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin: 5px;
+  border-radius: 3px;
+`;
+
+// Change colour of percent change based on increase/decrease/no change
+const Percent = styled.li`
+  color: ${(props) =>
+    props.isUp === 'up'
```

Comment:
FYI:
Not a big fan of nested ternaries, here's it's not too bad, but I've seen people to crazy nested ternaries to render different JSX and it gets really confusing.
Also, on some projects this rule is applied:
https://eslint.org/docs/rules/no-nested-ternary

One alternative would be to:
```
// In the styled component
const Percent = styled.li`
  color: ${(props) => props.color || 'darkslategrey'}); // ****
  ...

**** This seems a bit weird cos we're returning 'darkslategrey' as a default, but just as a failsafe, 
it's always good to add a backup in case the function gets broken for whatever reason.

// In the Body of the component
const getCorrectColor = (arrow) => {
  switch(arrow) {
    case 'up':
      return 'lime'
    case 'down':
      return 'red'
    default:
      return 'darkslategrey'
  }
}
  
// In the JSX
<Percent color={getCorrectColor(arrow)}>
```